.github/workflows/build-wheel-macos-arm64.yaml

```
@@ -259,40 +283,44 @@ jobs:
         cmake --build runtime-build --target rt_capi rtd_lightning rtd_openqasm
 
     - name: Test Catalyst-Runtime
+      env:
+        VENV_SITE_PACKAGES: ${{ steps.setup_env.outputs.venv_site_packages }}
       run: |
+        export PYTHONPATH="$VENV_SITE_PACKAGES:$PYTHONPATH"
         python${{ matrix.python_version }} -m pip install amazon-braket-pennylane-plugin
         cmake --build runtime-build --target runner_tests_lightning runner_tests_openqasm
         ./runtime-build/tests/runner_tests_lightning
         ./runtime-build/tests/runner_tests_openqasm
 
     # Build Quantum and Gradient Dialects
     - name: Build MLIR Dialects
-      run: |
+      run: |        
```

I would recommend enabling trailing white space stripping in your editor 😅 
```suggestion
      run: |
```